cypress/e2e/dashboard.cy.ts

```diff
@@ -0,0 +1,201 @@
+describe("Dashboard", () => {
+  type KeyType = {
+    [key: string]: string;
+  };
+
+  const keys: KeyType = {
+    loggedUserUnfundedPublicKey:
+      Cypress.env("LOGGED_USER_PUBLIC_KEY_UNFUNDED_TEST_ACCOUNT") || "",
+    loggedUserUnfundedSecretKey:
+      Cypress.env("LOGGED_USER_SECRET_KEY_UNFUNDED_TEST_ACCOUNT") || "",
+    loggedUserFundedPublicKey:
+      Cypress.env("LOGGED_USER_SECRET_KEY_FUNDED_TEST_ACCOUNT") || "",
+    loggedUserFundedSecretKey:
+      Cypress.env("LOGGED_USER_SECRET_KEY_FUNDED_TEST_ACCOUNT") || "",
```

Comment:
Just FUNDED_SOURCE_ACCOUNT_PUBLIC_KEY and the same with the other ones.

Reply:
I have taken your advice and used the names you suggested.